src/nnet3/nnet-utils.cc

```diff
@@ -629,14 +629,288 @@ void FindOrphanNodes(const Nnet &nnet, std::vector<int32> *nodes) {
   }
 }
 
+// this class implements the internals of the edit directive 'reverse-svd'
```

Comment:
I would prefer you to remoce the reverse-svd stuff before I merge.  You can always do this in a separate git branch from the one you are using internally for your work.

Reply:
Ok I removed the reverse svd part.